src/OrchardCore.Modules/OrchardCore.Workflows/Controllers/WorkflowTypeController.cs

```diff
@@ -63,7 +67,8 @@ public WorkflowTypeController
             ISecurityTokenService securityTokenService,
             IStringLocalizer<WorkflowTypeController> stringLocalizer,
             IHtmlLocalizer<WorkflowTypeController> htmlLocalizer,
-            IUpdateModelAccessor updateModelAccessor)
+            IUpdateModelAccessor updateModelAccessor,
+            JsonSerializerOptions jsonSerializerOptions)
```

Comment:
I get this and can't run the feature:

![image](https://github.com/OrchardCMS/OrchardCore/assets/1976647/59993d2b-9517-481d-9e8a-31282b383774)

Reply:
fixed

https://github.com/OrchardCMS/OrchardCore/assets/15613121/de60140c-4429-473b-9029-cb5dbcc1d8cf